publisher.rb

```diff
@@ -0,0 +1,38 @@
+require 'aws-sdk-sqs'
+class Publisher
+	# Note: You can store the values of the aws access keys in a config file (json) 
+	# from where you can get the values into the constants.
+	ACCESS_KEY_ID = #<your access key id>
+	SECRET_ACCESS_KEY = #<your secret access key>
+
+	attr_accessor :sqs 
+	def initialize
+		creds = Aws::Credentials.new(ACCESS_KEY_ID, SECRET_ACCESS_KEY)
+		@sqs = Aws::SQS::Client.new(region: 'ap-south-1', credentials: creds)
+	end
+
+	# For Standard Queue
+	def create_std_q queue_name
+		std_queue = sqs.create_queue(queue_name: queue_name)
+	end
+
+	# For FIFO Queue
+	def create_fifo_q queue_name
+		queue_name = queue_name.split('.')[0]
+		fifo_queue = sqs.create_queue(queue_name: "#{queue_name}.fifo", attributes: { "FifoQueue": "true" })
+	end
+
+	def publish_messages queue_url
+		999.times do |count|
```

Comment:
Don't have a restriction on the number of messages to publish. 
Create an infinite loop and the user can do `Ctrl + C` to exit the infinite loop